eng/pipelines/templates/jobs/archetype-sdk-client.yml

```diff
@@ -138,6 +156,9 @@ jobs:
     pool:
       vmImage: "$(OSVmImage)"
 
+    variables:
+      GeneratedPackageTargets: $[dependencies.Prepare.outputs['SelectPackages.GeneratedPackageTargets']]
```

Comment:
Do the variables that we generate end up getting scoped under the name of the step? I'm trying to understand why you did added `SelectedPackages.` to the front.

Reply:
Yeah - because we generate the variable in the Prepare job and publish it as an output variable, it gets scoped to the step that produced it. So ```Prepare``` refers to the job it came from and ```SelectPackages``` refers to the individual step.